### DISCUSSION TITLE
Release Kvrocks regularly like every 2 months?

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#1031] [General]
Currently, we don't have an explicit plan for when Kvrocks will publish the next release, even the community volunteers don't know when is ready. So it'd be better if we can have a rough plan like releasing every 2 or 3 months, then also most folks can be involved in the release voting process.

**Comment by PragmaTwice** [comment]
Good idea. We need more releases since user may not want to use the nightly version.
